src/py42/clients/_archive_access.py

```diff
@@ -278,11 +276,11 @@ def _wait_for_job(self, job_id):
             time.sleep(self._job_polling_interval)
 
     def _is_job_complete(self, job_id):
-        response = self._storage_archive_client.get_restore_status(job_id)
+        response = self._storage_archive_service.get_restore_status(job_id)
         is_done = response[u"done"]
         percentage_dict = {
             u"jobId": job_id,
-            u"status": response[u"status"],
+            u"status": response.data.get("status"),
```

Comment:
when I was testing this I saw that `status` was sometimes not in the response, particularly with smaller files, this prevents that from blowing up.

Reply:
Yay we can use `get()` now. Thank you Alan.

Reply:
Might be worth adding a test for this if you haven't already. I am fearful that this problem could potentially resurface.